data/euruko/euruko-2016/videos.yml

```diff
@@ -74,19 +74,29 @@
 - title: 'Lightning Talks: Day 1'
   raw_title: EuRuKo 2016 - Day 1 Lightning Talks
   speakers:
-    - Max Gorin # Making a Multiplayer Memoery Game with Elixir and Elm (0:29)
-    - Johanna Lang # Team Joda 2016 (Rails Girls Summer of Code 2016) (6:20)
-    - Dayana Mick # Team Joda 2016 (Rails Girls Summer of Code 2016) (6:20)
-    - Kinga Kalinowska # We Are Rubycats! (Rails Girls Summer of Code 2016) (12:11)
-    - Izabela Komorek # We Are Rubycats! (Rails Girls Summer of Code 2016) (12:11)
-    - Melanie Keatley # Magic Solution to the Tech Gender Gap (18:22)
-    - Stephanie Nemeth # Magic Solution to the Tech Gender Gap (18:22)
-    - Krzysztof (Christopher) Wawer # Validations (23:39)
-    - Demir Zekić # The Importance of Teaching and Metoring (28:53)
-    - Jan Lelis # Whirly - The Friendly Terminal Spinner (35:25)
+    - Max Gorin
+    - Johanna Lang
+    - Dayana Mick
+    - Kinga Kalinowska
+    - Izabela Komorek
+    - Melanie Keatley
+    - Stephanie Nemeth
+    - Krzysztof (Christopher) Wawer
+    - Demir Zekić
+    - Jan Lelis
```

Comment:
Thanks for adding the lightning talks into the description!

Can we keep the comment here? For the case where we want to implement #157 it would be great to have all the lightning talks in the same comment format so we can parse/search&replace them in one go!